test/TensorFlow/crashers.swift

```diff
@@ -248,8 +247,7 @@ public func testMultiResultOp_send_recv() {
   let results = TensorFlow.Raw.softmaxCrossEntropyWithLogits(features: x, labels: x)
   // Accelerator -> Host
   _hostOp(results.loss)
-  // expected-note @+2{{value used here}}
-  // expected-warning @+1{{implicitly copied to the accelerator}}
+  // expected-note @+1{{value used here}}
```

Comment:
can we remove `// TODO: The sends/recvs diagnostics are not very good. Fix them.` above now?

There might be some other related TODO/FIXME on sends/recvs diagnostics in the tests.

Reply:
Removed.

I searched all the tests that I modified and found another removable TODO, and removed that too.